includes/class-wc-stripe.php

```
@@ -791,6 +791,10 @@ protected function disable_upe( $settings ) {
 	 * @return WC_Email[]
 	 */
 	public function add_emails( $email_classes ) {
+		if ( ! class_exists( 'WC_Email', false ) ) {
+			return $email_classes;
```

PHPMD (2.15.0)

[error] The variable $email_classes is not named in camelCase. (undefined)

(CamelCaseVariableName)